### ISSUE TITLE
No such element error?

### ISSUE THREAD

**Issue by gowthamnatarajan** [amidst/toolbox#95]
![image](https://user-images.githubusercontent.com/5133248/35898099-59b7bf20-0b79-11e8-944d-b7bc53146b4f.png)

What does this error mean?

**Comment by andresmasegosa** [comment]
Not sure, unfortunately. Could you provide a piece of code and a data set to reproduce the bug? Then we can have a look and fix it.

**Comment by gowthamnatarajan** [comment]
The error is occurring while training. I am just calling updateModel.

Thew data is HUGE (nearly 30 GB). Does this mean a categorical variable  is taking a value which is not present in the Header in the ARFF file?

**Comment by andresmasegosa** [comment]
No, I don't think so. It seems related to sequence_id and time_id values of the instances. Could provide a small piece of the data (just few hundred samples) we could try to look at it? Otherwise, please check that your instances are grouped by sequence_id and then by time_id. And please also check that the provided sequence_ids and time_ids are consecutive. For example,

Seq_id, Time_id
0, 0
0, 1
0, 2
1, 0
1, 1
1, 2
1, 3
1, 4
2, 0
3, 0
3, 1
3, 2
....

**Comment by gowthamnatarajan** [comment]
Even the SEQUENCE_ID has to be in sequence? I just make sure that just TIME_ID is in sequence and not SEQUENCE_ID? 

I could have one SEQUENCE_ID missing in between.

**Comment by andresmasegosa** [comment]
You are right, SEQUENCE_ID no need to be in sequence.
